src/lib/api/ext/img-src.ts

```diff
@@ -57,12 +60,16 @@ export class ImgSrcDirective extends BaseFxDirective implements OnInit, OnChange
   @Input('src.gt-lg')  set srcGtLg(val) { this._cacheInput('srcGtLg', val);  }
   /* tslint:enable */
 
-  constructor(elRef: ElementRef,
-              renderer: Renderer2,
-              monitor: MediaMonitor,
-              @Inject(PLATFORM_ID) platformId: Object) {
-    super(monitor, elRef, renderer, platformId);
-    this._cacheInput('src', elRef.nativeElement.getAttribute('src') || '');
+  constructor(protected _elRef: ElementRef,
+              protected _monitor: MediaMonitor,
+              protected _styler: StyleService,
+              @Inject(PLATFORM_ID) protected _platformId: Object,
+              @Optional() @Inject(SERVER_TOKEN) protected _serverModuleLoaded: boolean) {
+    super(_monitor, _elRef, _styler);
+    this._cacheInput('src', _elRef.nativeElement.getAttribute('src') || '');
+    if (isPlatformServer(this._platformId) && this._serverModuleLoaded) {
```

Comment:
Can you explain a bit on how the serverModuleLoaded flag is used?

Reply:
The flag is to determine if the `FlexLayoutServerModule` is included (since it's the only module that provides that token).

Loading the server module is optional -- if a user doesn't load it, it means they want the inline styles as a fallback (it also prevents this from being a breaking change, since it's totally opt-in).

Reply:
Will this get into the docs?

Reply:
Absolutely. Docs will be in a follow-up PR once this has all been finalized